Containers/fulltextsearch/Dockerfile

```diff
@@ -1,15 +1,15 @@
 # Probably from here https://github.com/elastic/elasticsearch/blob/main/distribution/docker/src/docker/Dockerfile
 FROM elasticsearch:7.17.10
 
-RUN elasticsearch-plugin install --batch ingest-attachment
-
+ARG DEBIAN_FRONTEND=noninteractive

```

Comment:
```suggestion
```

Reply:
why not using ARG?

Reply:
because it is another step which could be done in the run step, no?

Reply:
using ARG/export don't change the end result, but I prefer ARG, because then I have a list of build time envs at the top of the file